### PR TITLE
OSDOCS-3341: Move IBM API key rotation procedure

### PR DESCRIPTION
**Pull request by jeana-redhat**
For [OSDOCS-3341](https://issues.redhat.com/browse/OSDOCS-3341)

Previews: 
- New section **Rotating or removing cloud provider credentials >** [**Rotating cloud provider credentials with the Cloud Credential Operator utility**](https://deploy-preview-43058--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#ccoctl-rotate-remove-cloud-creds) in **Post-installation cluster tasks**. Currently it just holds the IBM-specific procedure, but we can use this heading to add other platforms as needed.
- Added additional resources under [**Configuring the Cloud Credential Operator utility**](https://deploy-preview-43058--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/configuring-iam-ibm-cloud.html#cco-ccoctl-configuring_configuring-iam-ibm-cloud) to call users' attention to this capability.

### CODE REVIEW

**Comment by jeffnowicki** [comment]
@jeana-redhat Are you actually moving/relocating that documentation or simply adding it to the alternate area? I personally like having it where it is, but I'm ok with also having it in the new location. If we can have it in 'both' locations that would be good. If you have to pick one, I'll defer to your recommendation.

**Comment by jeana-redhat** [comment]
@jeffnowicki yes, I planned to move it, parallel to the placement for other platforms. However, what do you think about adding a link to its new home in an "Additional references" section following its [current location](https://docs.openshift.com/container-platform/4.10/installing/installing_ibm_cloud_public/configuring-iam-ibm-cloud.html#refreshing-service-ids-ibm-cloud_configuring-iam-ibm-cloud)?

**Comment by jeffnowicki** [comment]
@jeana-redhat I like your suggestion.

**Comment by jeana-redhat** [comment]
@jianping-shu this ideally should not have any technical impact (just moving content around), but should still get an ack from QA. PTAL when you can :)

File: post_installation_configuration/cluster-tasks.adoc
Comment (jianping-shu) [inline post_installation_configuration/cluster-tasks.adoc:611]: typo 'taht'

**Comment by jianping-shu** [comment]
I have a comment which is not related to the ticket content.
When I looked at the chapter of Configuring IAM for IBM Cloud, it is a little weird that "Additional resources" only lists "Rotating API keys for IBM Cloud". Is it better to list all the other resources like  "Manually creating IAM for IBM Cloud"?

**Comment by jeana-redhat** [comment]
Thanks Jianping! In this case, I think just one resource is fine (it is allowed, even though it says "resources" plural :slightly_smiling_face:).  They will see steps to create IAM in the context of installing, when they need to do it, so I don't think it adds context here. 

Fixed the typo in the comments and refreshing the PR.

**Review by adellape — APPROVED** [review]

**Pull request merged**
